cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -484,6 +576,498 @@ def test_frames_outside_are_not_generated(self):
+                        zip_ref.extractall(folder_name)
+                    data_datumaro[type_file] = datumaro.components.project.Dataset.import_from(folder_name, importer_format_name[dump_format_name])
+
+                # equals dataset vs annotations
+                compare_datasets(self, data_datumaro["dataset"], data_datumaro["annotations"])
+
+    @skip("Fail")
+    def test_api_v1_tasks_annotations_dump_and_upload_with_datumaro_many_jobs(self):
+        test_name = self._testMethodName
+        # create task with annotations
+        for dump_format_name in ["CVAT for images 1.1", "CVAT for video 1.1"]:
+            images = self._generate_task_images(13)
+            task = self._create_task(tasks["many jobs"], images)
+            self._create_annotations(task, f'{dump_format_name} many jobs', "dafault")
+
+            task_id = task["id"]
+            task_ann = TaskAnnotation(task_id)
+            task_ann.init_from_db()
+            task_data = TaskData(task_ann.ir_data, Task.objects.get(pk=task_id))
+            extractor = CvatTaskDataExtractor(task_data)
+            data_from_task_before_upload = datumaro.components.project.Dataset.from_extractors(extractor)
+
+            # dump annotations
+            url = f"/api/v1/tasks/{task_id}/annotations"
+            data = {
+                "format": dump_format_name,
+                "action": "download",
+            }
+            file_zip_name = osp.join(osp.dirname(__file__), 'assets', f'{test_name}_{dump_format_name}.zip')
+            self._download_file(url, data, self.admin, file_zip_name)
+            self._check_downloaded_file(file_zip_name)
+
+            # remove annotations
+            response = self._delete_request(url, self.admin)
+            self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+            # upload annotations
+            if dump_format_name in ["CVAT for images 1.1", "CVAT for video 1.1"]:
+                upload_format_name = "CVAT 1.1"
+            else:
+                upload_format_name = dump_format_name
+
+            url = f"/api/v1/tasks/{task_id}/annotations?format={upload_format_name}"
+            with open(file_zip_name, 'rb') as binary_file:
+                response = self._upload_file(url, binary_file, self.admin)
+                self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED)
+
+            # equals annotations
+            task_ann = TaskAnnotation(task_id)
+            task_ann.init_from_db()
+            task_data = TaskData(task_ann.ir_data, Task.objects.get(pk=task_id))
+            extractor = CvatTaskDataExtractor(task_data)
+            data_from_task_after_upload = datumaro.components.project.Dataset.from_extractors(extractor)
+            compare_datasets(self, data_from_task_before_upload, data_from_task_after_upload)
+
+    def test_api_v1_tasks_annotations_dump_and_dataset_export_empty_data_with_datumaro(self):
```

Comment:
We have a similar test below in the file.

Reply:
OK. Removed this test.